spec/server.test.js

```diff
@@ -4,7 +4,7 @@ const app = require('../server/app.js');
 
 
 xdescribe('should send back a 200 if static file is found', () => {
-  test('It should respond with a 200', (done) => {
+  xtest('It should respond with a 200', (done) => {
```

Comment:
I don't think the `xtest()` is necessary here, the `xdescribe()` should skip everything inside it